src/aks-preview/HISTORY.rst

```diff
@@ -15,6 +15,7 @@ Pending
 * Add `az aks nodepool delete-machines` command.
 * Update `az aks approuting zone` command to support private dns zones.
 * Vendor new SDK and bump API version to 2024-01-02-preview.
+* Add `az aks check-network` command.
```

Comment:
please rebase from main and put your history note under a new version number (update `VERSION` in setup.py accordingly) or Pending

Reply:
I thought I put it under Pending but probably because of conflict it ended up in the wrong place. Updated to put under Pending.